### DISCUSSION TITLE
Nebula Operator Storaged multiple dataVolumeClaims

### DISCUSSION THREAD

**Discussion by Sahil333** [vesoft-inc/nebula#5982] [Q&A]
In nebula-operator's nebulacluster definition, what is the use/purpose of multiple dataVolumeClaims in storaged unlike single dataVolumeClaim in metad?

**Comment by xiajingchun** [comment]
This has to do with the data partitioning capability in nebula. 
In storaged, normally you will have more than one partitions to store graph data, and storaged is able to distribute the partitions into multiple `data_paths`, so multiple dataVolumeClaims can be defined when necessary.
In metad, however, there's alway one partition to store the cluster metadata, so only one dataVolumeClaim suffices.
